Authentication/AuthenticationBootstrapper.swift

```diff
@@ -145,16 +145,35 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
     public func createRecoverPasswordController() -> RecoverPasswordController { //todo
         return RecoverPasswordController()
     }
-    
-}
-
-private extension AuthenticationBootstrapper {
 
     func createLoginController() -> LoginController {
-        return LoginController(
+        let configuration = LoginControllerConfiguration(
             viewModel: createLoginViewModel(),
-            loginViewFactory: createLoginView,
-            delegate: createLoginControllerDelegate())
+            viewFactory: createLoginView,
+            transitionDelegate: self)
+        return LoginController(configuration: configuration)
     }
+    
+}
 
+public extension AuthenticationBootstrapper {
```

Comment:
When you conform to protocols in extension declare the protocol in the extension

```swift
public extension AuthenticationBootstrapper: LoginControllerTransitionDelegate
```